src/views/zesty/Homepage/index.js

```diff
@@ -82,6 +83,13 @@ const GetDemoSection = dynamic(() => import('./GetDemoSection'), {
 function Homepage({ content }) {
   const { palette } = useTheme();
   const isLoggedIn = useIsLoggedIn();
+  const zestyProductionMode = getCookie('PRODUCTION');
+
+  useEffect(() => {
+    if (!zestyProductionMode) {
+      setCookie('PRODUCTION', content?.zestyProductionMode);
```

Comment:
What is the purpose of setting another cookie here? Looks like this can be derived from the page URL already.